test/Sentry.Tests/GlobalSessionManagerTests.cs

```diff
@@ -292,12 +294,13 @@ public void TryGetPersistentInstallationId_SessionStarted_ReturnsLastSession()
         var sut = _fixture.GetSut();
 
         var sessionUpdate = sut.StartSession();
-
+        _fixture.Clock.GetUtcNow().Returns(DateTimeOffset.Now.AddSeconds(1));
```

Comment:
It will still be broken if it runs with 0 ticks, as both calls would return the same value (whatever + 1)

Reply:
keep the counter in scope, and `++` on it so each call to the method increments the value